dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -0,0 +1,181 @@
+                                    <controls:NumberBox x:Name="NumberBoxDisplay"
+                                                    HorizontalAlignment="Center"
+                                                    VerticalAlignment="Center"
+                                                    Maximum="{Binding RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource IntToDouble}, Path=NumberOfPages}"
+                                                    Minimum="1"
+                                                    Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource IntToDouble}, Path=SelectedIndex, Mode=TwoWay}"
+                                                    SmallChange="1"
+                                                    LargeChange="5"
+                                                    IsEnabled="False"
+                                                    Visibility="Collapsed" />
+                                    <!--ComboBox Display-->
+                                    <ComboBox x:Name="ComboBoxDisplay"
+                                          HorizontalAlignment="Center" VerticalAlignment="Center"
+                                          SelectedIndex="{Binding RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource IntMinusOne}, Path=SelectedIndex, Mode=TwoWay}"
+                                          ItemsSource="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=PagerTemplateSettings.Pages}"
+                                          IsEnabled="False"
+                                          Visibility="Collapsed"/>
+                                    <TextBlock VerticalAlignment="Center" Margin="5,0,5,0" Text="of"/>
+                                    <TextBlock VerticalAlignment="Center" Margin="0,0,5,0" Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource IntToStr}, Path=NumberOfPages}"/>
+                                </StackPanel>
+                                <StackPanel x:Name="NumberPanelDisplay"
+                                            Orientation="Horizontal"
+                                            Visibility="Collapsed"
+                                            Grid.Row="0" Grid.Column="1">
+                                    <ScrollViewer x:Name="NumberPanelViewer"
+                                              HorizontalScrollBarVisibility="Hidden" HorizontalScrollMode="Disabled"
+                                              VerticalScrollBarVisibility="Disabled" VerticalScrollMode="Disabled">
+                                        <controls:ItemsRepeater x:Name="NumberPanelItemsRepeater"
+                                        ItemsSource="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=PagerTemplateSettings.Pages}"
+                                        ItemTemplate="{StaticResource IntToButton}">
```

Comment:
>IntToButton [](start = 70, length = 11)

The name here could be more descriptive maybe NumberPanelButtonTemplate

Reply:
Agreed. I have changed it